src/nodes.coffee

```diff
@@ -535,6 +538,17 @@ exports.NaNLiteral = class NaNLiteral extends NumberLiteral
 
 exports.StringLiteral = class StringLiteral extends Literal
 
+  compileNode: (o) ->
+    res = if @csx then [@makeCode @unquote yes] else super()
+
+  unquote: (literal) ->
+    unquoted = @value[1...-1]
+    if literal
+      unquoted.replace /\\n/g, '\n'
```

Comment:
@xixixao Could you please reply to this, and also explain what’s going on with the escaping here. Per @lydell’s earlier comment, why is the escaping so simple? Why can we ignore things like escaped backslashes?